src/common/header/components/Banner.tsx

```diff
@@ -28,12 +30,23 @@ function TipNav({ onClick, icon }: { onClick: () => void; icon: JSX.Element }):
 export function Banner(): JSX.Element {
   const [banner, setBanner] = useSessionStorage('user:banner', true);
   const [tipIndex, setTipIndex] = useState(0);
+  const { term } = useParams();
 
   const tips: Array<JSX.Element> = [
     <Text>💡 Your courses and sections are saved between sessions, no need to leave the tab open!</Text>,
     <Text>
-      💡 Press the <Text as="strong">Register</Text> button while viewing your timetable to help you quickly register
-      for classes!
+      💡 Press the{' '}
+      <Button
+        size="xs"
+        mx="1"
+        mt="-1"
+        colorScheme="blue"
+        as={Link}
+        to={term ? `/registration/${term}` : `/registration/${getCurrentTerm()}`}
+      >
+        Register
+      </Button>{' '}
+      button while viewing your timetable to help you quickly register for classes!
```

Comment:
I'm kinda for not having it functional just cause being taken there off while in timetable view etc. and switching terms is a bad idea. Making that work seamlessly seems like too much work so.